argus/backend/controller/testrun_api.py

```diff
@@ -289,6 +289,22 @@ def ignore_jobs():
     }
 
 
+@bp.route("/resolve_runs", methods=["POST"])
+@api_login_required
+def resolve_runs():
```

Comment:
why not `get_runs_build_id_and_number`?
For me would be much easier to understand

Reply:
Done